spao/urls.py

```diff
@@ -13,10 +13,11 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
-from django.contrib import admin
-from django.urls import path
-from django.urls.conf import include
+from django.urls import path, include
 
 urlpatterns = [
-    path('users', include('users.urls')),
+
+    path('users',    include('users.urls')),
+    path('products', include('products.urls')),
+
```

Comment:
- 불필요한 공백 없애주세요.
- 함수의 parameter 부분은 굳이 정렬 안해주셔도 괜찮습니다.
```suggestion
    path('users', include('users.urls')),
    path('products', include('products.urls')),
```

Reply:
수정했습니다